test/translator/inquiry_test.cc

```diff
@@ -284,6 +285,76 @@ TEST(TranslateUnitSerialNumberVpd, both) {
   ASSERT_STREQ((char *)product_serial_number, formatted_hex_string);
 }
 
+TEST(TranslateIdentificationDescriptorNGUID, ReturnsCorrectFields) {
```

Comment:
Perhaps a better test naming convention could be to name the test suite name (first argument) after the method being tested, and the second argument (test name) describing the specifics about what the test is testing. `ReturnsCorrectFields ` is not very descriptive as a test name.

Reply:
Yup, I agree. I made the test names more descriptive